SolutionArchitecture.md

```diff
@@ -0,0 +1,101 @@
+### **Solution Architecture Walkthrough**
+
+![image](/images/readMe/image4.png "Inserting image...")
+
+Call and text recordings are uploaded to the web application. Those
```

Comment:
Not uploaded to the web application, but uploaded through or via the web application